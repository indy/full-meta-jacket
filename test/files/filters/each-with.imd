---
"posts": [{"title": "alpha"}, {"title": "beta"}, {"title": "delta"}]
---
{{=: posts | eachWith:'list-item'}}
