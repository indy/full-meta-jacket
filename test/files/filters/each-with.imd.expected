<li>alpha</li><li>beta</li><li>delta</li>